docs/guides/quality.md

```diff
@@ -1,142 +1,189 @@
-| Dual Audio      | Has two audio tracks. Typically English and Japanese              |
-| [F0EAA72E]      | Random string at the end of the filename is typically CRC32.      |
+===
 
-### Quality
+### Encoding
 
-Quality might be somewhat subjective. The best release for the purpose of this guide is an encode which fixes flaws of the raw bdmv while simultaneously not altering the original material too much. There is a difference between obvious flaws which need to be fixed and your subjective preference of how you like the encode. If something apparently looks "better" because it unnecessarily alters the video, then it's not a good indication of quality. These statements might seem too complicated or a bit vague, I'll try to explain in brief, but this is beyond the scope of a guide for beginners and more into the realm of encoding.
+Quality is subjective to an extent, and there is a difference between flaws that need to be fixed and the preferences of the end user.
 
-The official BDMV, from the blurays, is itself an encode of the actual source. It often suffers from issues like banding, blocking, noise, aliasing, etc. These might be originating from a variety of reasons. For example, a lot of anime is native 720p and is upscaled to 1080p for the bluray. Even if issues aren't present in the bdmv, they can appear in the encode because of compression. This fixing is called filtering, a step which comes before encoding. The video is filtered with Avisynth/Vapoursynth before passing it on to the encoder.
+Generally, the best release is an encode which fixes the flaws of a raw BDMV while simultaneously not altering the original material. There are objective flaws called **artifacts**, a term used to broadly describe defects or foreign, unwanted elements in a video. There can be a number of causes, ranging from lossy compression, improper conversions, to post-processing adjustments like sharpening and resampling. *You can read about artifacts in detail [here](https://guide.encode.moe/encoding/video-artifacts.html).* On the other hand, end user preferences like excessive sharpening or contrast are destructive and go against the idea of a good encode.
+
+The official BDMV is an encode of the original source master. Subsequently, it often suffers from various issues (e.g. aliasing, banding, blocking, noise) originating from a variety of reasons. For example, a lot of anime is natively 720p and is upscaled to 1080p for the Blu-ray. *Even if issues aren't present in the BDMV, they can appear in the encode due to compression.*
+
+These issues can be fixed through filtering, a step which comes before encoding. The video is filtered with tools like [Vapoursynth](https://www.vapoursynth.com) before passing it on to the encoder. Since encoding is a lossy process, filtering is necessary to improve the video quality along with transparent encoder settings. *As a result, almost all unfiltered encodes are worse than the source.*
```

Comment:
replace the site link with GitHub repo instead https://github.com/vapoursynth/vapoursynth due to the site not being as reliable

Reply:
Added in https://github.com/Snaacky/thewiki/pull/86/commits/3120f9c5c712daa073cf27d30cdeafbb9efe25c5.